packages/components/pie-switch/test/visual/pie-switch.spec.ts

```diff
@@ -41,10 +42,8 @@ test.describe('Prop: `Label`', () => {
 
 test.describe('when in RTL settings', () => {
     [true, false].forEach(async (checkedState) => {
-        test(`should render in rtl correctly when (checked: ${checkedState})`, async ({ page, mount }) => {
-            await page.evaluate(() => {
-                document.documentElement.setAttribute('dir', 'rtl');
-            });
+        test.only(`should render in rtl correctly when (checked: ${checkedState})`, async ({ page, mount }) => {
```

Comment:
Did you mean to leave the `.only` in?

Reply:
Ooops. Yeah I'll remove that. Forgot about that.